### PR TITLE
added quotes to TARGET_PATH variable

### PR DESCRIPTION
**Pull request by monotek**
### Type of Change

- [x] Bugfix
- [ ] Enhancement / new feature
- [ ] Refactoring
- [ ] Documentation

### Description

* fixes cla link
* fixes variable without qoutes

```
shellcheck entrypoint.sh 

In test.sh line 16:
yamllint -f parsable ${TARGET_PATH} | /yamllint-action
                     ^------------^ SC2086: Double quote to prevent globbing and word splitting.

```

### Checklist

- [ ] Write tests
- [ ] Make sure all tests pass
- [ ] Update documentation
- [ ] Review the [Contributing Guideline](https://github.com/Staffbase/yamllint-action/blob/master/CONTRIBUTING.md) and sign CLA
- [ ] Reference relevant issue(s) and close them after merging

### CODE REVIEW

**Review by 0x46616c6b — APPROVED** [review]

File: entrypoint.sh [hunk 1]
```diff
@@ -13,4 +13,4 @@
 
 set -e
 
-yamllint -f parsable ${TARGET_PATH} | /yamllint-action
+yamllint -f parsable "${TARGET_PATH}" | /yamllint-action
```
Comment (monotek) [inline entrypoint.sh:16]: I have read the CLA Document and I hereby sign the CLA

**Comment by monotek** [comment]
recheck

**Comment by monotek** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by monotek** [comment]
recheck

**Comment by axdotl** [comment]
comment for test

**Comment by axdotl** [comment]
recheck

**Comment by axdotl** [comment]
recheck

**Pull request merged**